src/global/Constants.h

```diff
@@ -8,25 +8,24 @@
 
 #include "../util/Parameters.h"
 
-static const size_t DEFAULT_STXXL_MEMORY_IN_BYTES =
-    1024UL * 1024UL * 1024UL * 5UL;
-static const size_t STXXL_DISK_SIZE_INDEX_BUILDER = 1000 * 1000;
+static const size_t DEFAULT_STXXL_MEMORY_IN_BYTES = 5'000'000'000UL;
+static const size_t STXXL_DISK_SIZE_INDEX_BUILDER = 1000;  // In MB.
 static const size_t STXXL_DISK_SIZE_INDEX_TEST = 10;
 
 static constexpr size_t DEFAULT_MEM_FOR_QUERIES_IN_GB = 4;
 
-static const size_t MAX_NOF_ROWS_IN_RESULT = 100000;
+static const size_t MAX_NOF_ROWS_IN_RESULT = 1'000'000;
```

Comment:
I am currently not sure whether this constant is even active with the latest changes to the Sparql parser, but was the change from 100K to 1M here deliberate?

Reply:
It was deliberate, sorry for not mentioning it explicitly. I was even wondering whether we need this at all. It's a rather random protection, is it not? For an effective DOS protection, we would have to do much more.